News/Services/UserService.cs

```diff
@@ -30,25 +30,48 @@ public async Task<UserResponseModel> GetUserById(Guid id)
         return result;
     }
 
+    public async Task<UserResponseModel> Login(LoginModel login)
+    {
+        var user = await _repo.GetUserByLogin(login.Login);
+        if (user == null)
```

Comment:
better to check for null using "if (user is null) {}"

Reply:
Why double eq is worse?

Reply:
Because the == operator can be overloaded, it might be difficult for me to explain. It would be better if you searched for information on the difference between "==" and "is" when checking an object for "null".